### DISCUSSION TITLE
ScriptReference/GL.LoadPixelMatrix

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#190] [Notes]
# ScriptReference/GL.LoadPixelMatrix

https://docs.unity3d.com/ScriptReference/GL.LoadPixelMatrix.html

**Comment by mvi** [comment]
Note that the z value is inclusive of +1 on Windows, but exclusive on Mac. If you wish to draw at the very front you should use a value of 0.99 instead of 1 to maintain cross platform support. Otherwise your polygon will render correctly on Windows but be invisible on Mac.
